extras/.vsconfig

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
```

Comment:
One thing that concerns me is that this does not specify versions, I assume it chooses the latest. This could cause very hard to diagnose issues.

The full list of component IDs is available here: https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-community?view=vs-2019

Reply:
I think this will be fine for now as long as we explicitly reference the versions in the readme. Perhaps the readme should instruct to check the selected components match the versions in the readme..